src/server/service/kv.rs

```diff
@@ -1132,14 +1133,16 @@ fn handle_batch_commands_request<E: Engine, L: LockManager>(
 fn future_handle_empty(
     req: BatchCommandsEmptyRequest,
 ) -> impl Future<Item = BatchCommandsEmptyResponse, Error = Error> {
-    tikv_util::timer::GLOBAL_TIMER_HANDLE
-        .delay(std::time::Instant::now() + std::time::Duration::from_millis(req.get_delay_time()))
-        .map(move |_| {
-            let mut res = BatchCommandsEmptyResponse::default();
-            res.set_test_id(req.get_test_id());
-            res
-        })
-        .map_err(|_| unreachable!())
+    let mut res = BatchCommandsEmptyResponse::default();
+    res.set_test_id(req.get_test_id());
+    if req.get_delay_time() < 10 {
```

Comment:
Better to add a comment here. BTW, is this change will affect existing test?

Reply:
I'm not sure what tests are using the message. @hicqu do you have any ideas?

Even they exist, I doubt whether those tests are working correctly.